### PR TITLE
chore: skip devDep on audit check

### PR DESCRIPTION
**Pull request by lcovar**
TICKET: BG-0000

### CODE REVIEW

**Review by ericcrosson-bitgo — APPROVED** [review]
Might need to run `npm install` and commit the package-lock.json changes too

**Review by ericcrosson-bitgo — COMMENTED** [review]
Recall also that semantic-release@20 requires Node.js 18, which you can specify here:

https://github.com/BitGo/bitcoinjs-lib/blob/9441d18e5b4eff56631129e1507ed89dd88007e5/.github/workflows/main_ci.yml#L145

File: .nsprc [hunk 1]
```diff
@@ -1 +1,7 @@
-{}
+{
+  "GHSA-rc47-6667-2j5j": {
+    "active": true,
+    "notes": "ignore until a fix is introduced to semantic-releases",
```
Comment (louib-bitgo) [inline .nsprc:4]: @lcovar You could also add a link to [this issue](https://github.com/semantic-release/npm/issues/574) in the notes.

Comment (lcovar) [reply .nsprc]: added, ty

**Review by davidkaplanbitgo — APPROVED** [review]

**Review by louib-bitgo — APPROVED** [review]

**Pull request merged**